python/interpret_community/explanation/explanation.py

```diff
@@ -1661,17 +1661,17 @@ def _get_local_explanation_row(explainer, evaluation_examples, i, batch_size):
     return explainer.explain_local(rows)
 
 
-def _aggregate_streamed_local_explanations(explainer, evaluation_examples, classification, features,
+def _aggregate_streamed_local_explanations(explainer, evaluation_examples, task, features,
```

Comment:
this seems a breaking change, do you mind check if any of our notebook or reference code in rai or aml does not have named parameter?

Reply:
It's not a breaking change, we are still passing the same parameter value, just the variable name that is passed was changed.  Regardless, this method is private because it begins with an underscore anyway, so we can change it however we like.

Reply:
It's not an optional keyword argument, right?